Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM cypress/browsers:chrome69
```

Comment:
```suggestion
ARG BROWSER="chrome69"
FROM cypress/browsers:$BROWSER
```

Reply:
Would allow us to build an image for multiple browsers

Reply:
Yeah, I love the suggestion 😄